<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWindow</class>
 <widget class="QMainWindow" name="TestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_img_logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[Img] Logo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_img_logo_neg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[Img] Logo neg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_img_img1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[Img] Img 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_img_img2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[Img] Img 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_vid_long">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[Vid] Vid long</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_vid_medium">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[Vid] Vid medium</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_vid_short">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[Vid] Vid short</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_vid_extra">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[Vid] Extra</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
